addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -455,7 +456,7 @@ def test_channel_command_help_in_group(self):
                     "payload": {
                         "body":
                             "<span class='o_mail_notification'>"
-                            "You are in a private conversation with <b>@Mitchell Admin</b> and <b>@&lt;strong&gt;Evita Employee NoEmail&lt;/strong&gt;</b>."
+                            f"You are in a private conversation with <b>@{self.user_admin.partner_id.name}</b> and <b>@&lt;strong&gt;{original_partner_employee_nomail_name}&lt;/strong&gt;</b>."
```

Comment:
Rather than making the name dynamic, we should ensure it takes the one we expect

Reply:
The failure was
```
Expected:
{"type":"discuss.channel/transient_message","payload":{"body":"<span class='o_mail_notification'>You are in a private conversation with <b>@Mitchell Admin</b> and <b>@&lt;strong&gt;Evita Employee NoEmail&lt;/strong&gt;</b>.<br><br>Type <b>@username</b> to mention someone, and grab their attention.<br>Type <b>#channel</b> to mention a channel.<br>Type <b>/command</b> to execute a command.</span>","originThread":{"model":"discuss.channel","id":145}}}
Returned:
{"type":"discuss.channel/transient_message","payload":{"body":"<span class='o_mail_notification'>You are in a private conversation with <b>@Administrator</b> and <b>@&lt;strong&gt;Evita Employee NoEmail&lt;/strong&gt;</b>.<br><br>Type <b>@username</b> to mention someone, and grab their attention.<br>Type <b>#channel</b> to mention a channel.<br>Type <b>/command</b> to execute a command.</span>","originThread":{"model":"discuss.channel","id":145}}}
```

So I think it is because the `no demo nightly build` has a different naming in Admin?